api/db-cdr/generate-cdr/libproject/genomic_manifests.rb

```diff
@@ -605,7 +691,7 @@ def _maybe_update_preprod_access(project, all_tasks, add)
   if needs_preprod_grant
     preprod_source_projects.each do |p|
       _update_project_iam_object_viewer(preprod_appspot_account, preprod_deploy_account, p, add)
-      common.status "Sleeping for 1m after IAM grant to mitigate consistency issues"
+      Common.new.status "Sleeping for 1m after IAM grant to mitigate consistency issues"
```

Comment:
what does this change do?

Reply:
common was not defined. Common has to be initialized for this to work. Check other usages in the same method.